lib/cocoapods-packager/builder.rb

```diff
@@ -110,9 +110,16 @@ def copy_headers
       headers_source_root = "#{@public_headers_root}/#{@spec.name}"
 
       Dir.glob("#{headers_source_root}/**/*.h").
-        each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
-
-      if File.exist?("#{@public_headers_root}/#{@spec.name}/#{@spec.name}.h")
+          each { |h| `ditto #{h} #{@fwk.headers_path}/#{h.sub(headers_source_root, '')}` }
+
+      # if custom 'module_map' is specified add it
+      # to the framework distribution, otherwise check
+      # if a header exists that is equal to 'spec.name'', if so
+      # create a default 'module-map' one using it.
```

Comment:
Please apply some minor formatting changes to the comment:
- let use at least 80 chars per line before wrapping
- consistently use 'module_map'
- drop the second ' after spec.name
- start the comment with a capital 'i'